modules/howtos/partials/acid-transactions-attributes.adoc

```diff
@@ -0,0 +1,23 @@
+// These attributes are used in sdk::shared:partial$acid-transactions.adoc 
+
+// intro
+:durability-exception: pass:q[`DurabilityImpossibleException`]
```

Comment:
Ah yes, as discussed before with Richard, this looks quite in-depth!
Only concern would be it might be a little fragile?

(But is fine, let's see how it pans out 👍)

Reply:
Think as long as we are sensible with how we do this, it should be fine.